### DISCUSSION TITLE
Steps and scenarios skipped in test

### DISCUSSION THREAD

**Discussion by oappere** [cucumber-rs/gherkin#44] [Q&A]
Hi
When I launch tests ("cargo test") tests pass but the steps are indicated as skipped and a question mark appears. 
Why is that ? Looks like whatever is written after is discarded.

```
Funktion: Test
  Beispiel: Scenario
   ?  Angenommen given
      Step skipped: tests/features/de.feature:5:5
Feature: Everything
  Scenario: Nothing
   ?  Given I just started
      Step skipped: tests/features/main.feature:4:3
Egenskap: Gjett et ord
  Eksempel: Ordmaker starter et spill
   ?  Når Ordmaker starter et spill
      Step skipped: tests/features/no.feature:7:5
```

**Comment by tyranron** [comment]
@oappere it seems that no step matching functions were discovered.  Could be the same reason as https://github.com/cucumber-rs/cucumber/issues/347: `inventory` cannot find them. Try to define your `World` and step matching functions in the same crate.
